Lib/trace.py

```diff
@@ -385,14 +385,16 @@ def _find_executable_linenos(filename):
     return _find_lines(code, strs)
 
 class Trace:
-    def __init__(self, count=1, trace=1, countfuncs=0, countcallers=0,
+    def __init__(self, count=1, trace=1, trace_fd=1, countfuncs=0, countcallers=0,
```

Comment:
Adding a parameter before other positional parameters is a breaking change. It will break a code which calls Trace() with 3 or more positional arguments.

Add it at the end of parameters, and better make it keyword-only. 10 positional parameters is too much.

Also not that no other parameter has `_` in the name, even if they are a combination of words.

Reply:
Ok, fixing this.